assets/app/features/upgrade/upgrade.constants.js

```diff
@@ -8,5 +8,9 @@
             'openstack': ['SUSE-OpenStack-Cloud-7-Pool', 'SUSE-OpenStack-Cloud-7-Updates'],
             'ceph': ['SUSE-Enterprise-Storage-4-Pool', 'SUSE-Enterprise-Storage-4-Updates']
         })
+        .constant('ADDONS_PRECHECK_MAP', {
+            'ha': ['clusters_healthy'],
```

Comment:
Seems sensible to me to have an array in case we introduce further checks per addon in the future.